spec/requests/api/v1/notifications_spec.rb

```diff
@@ -6,12 +6,17 @@
   let!(:admin_user) { create(:confirmed_user, admin: true) }
   let!(:logged_in_user) { create(:confirmed_user, :nebraska) }
   let!(:business) { create(:business, :nebraska_ldds, user: logged_in_user) }
-  let!(:approval) { create(:approval, num_children: 3, business: business) }
-  let!(:children) { approval.children }
+  let!(:approval) { create(:approval, num_children: 1, business: business, expires_on: 3.days.after) }
+  let!(:early_approval) { create(:approval, num_children: 1, business: business, expires_on: 1.day.after) }
+  let!(:late_approval) { create(:approval, num_children: 1, business: business, expires_on: 29.days.after) }
+  let!(:children) { [approval.children.first, early_child, late_child] }
+  let!(:early_child) { early_approval.children.first }
+  let!(:late_child) { late_approval.children.first }
   let!(:non_owner_child) { create(:necc_child) }
 
   before do
-    create(:notification, child: children.first, approval: approval)
+    approvals = [approval, early_approval, late_approval]
```

Comment:
This creates three notifications with three different approval expiry times that are intentionally in the wrong order.